roda-ui/roda-wui/src/main/java/org/roda/wui/client/browse/bundle/BrowseAIPBundle.java

```diff
@@ -25,6 +25,7 @@ public class BrowseAIPBundle implements Bundle {
   private Long riskIncidenceCount;
   private Long preservationEventCount;
   private Long logCount;
+  private String instanceName;
```

Comment:
Add this attribute in the toString method of this class

Reply:
Done